### DISCUSSION TITLE
Faster Deploys?

### DISCUSSION THREAD

**Discussion by eroltoker** [brefphp/bref#1859] [Support]
Hello,

Is it possible to make use of serverless function to deploy only updates to the function code/zip file, such as this:
sls deploy function --function api --stage dev --debug

I'm already on brefv2 and have the following set up in my serverless.yml file:
    deploymentMethod: direct # fastest deployment method

However it still takes about a minute and right now i run two different services/bref apps that deploy from a monorepo so it's in fact 2 minutes to get a deploy going. 

Appreciate any tips!

**Comment by mnapoli** [comment]
I wouldn't recommend that technique in prod, but in dev/staging yes you can definitely do it.
